scripts/release/steps/update-version.js

```diff
@@ -24,6 +24,8 @@ export default async function updateVersion({ version }) {
     ),
   );
 
+  await runYarn(["install"], { cwd: "./website" });
```

Comment:
Let's use absolute path here?

Reply:
I don't mind that. Release script is executed in repository root.